Library/Formula/zeromq.rb

```diff
@@ -1,8 +1,6 @@
 class Zeromq < Formula
   homepage "http://www.zeromq.org/"
-  url "http://download.zeromq.org/zeromq-4.0.5.tar.gz"
-  sha1 "a664ec63661a848ef46114029156a0a6006feecd"
-  revision 2
+  revision 3
```

Comment:
It's probably worthwhile. I think I recall Mike bumped the revision last because he thought he'd removed the ` libsodium ` mandatory dep, but it's a solid bet for a decent chunk of people they still have it with ` libsodium ` but aren't aware. Bumping the revision again now we know 100% that this will not impact anyone else unless they explicitly build with ` libsodium ` is probably worth the hassle. Just my :dollar:.